library/spec/Ivoz/Provider/Domain/Service/BillableCall/UpdateDtoByDefaultRunTpCdrSpec.php

```diff
@@ -146,6 +146,10 @@ private function prepareExecution()
             BillableCallDto::class
         );
 
+//        $this->billableCallDto
+//            ->getStartTime()
+//            ->willReturn(new \DateTime('2000-01-01 00:00:00'));
+
```

Comment:
Commented code is not allowed

Reply:
This code belongs to the parent branch where it is already fixed.
I have rebased both branches and this commit is gone.